lib/gds_api/publishing_api_v2.rb

```diff
@@ -1,6 +1,14 @@
 require_relative 'base'
 
 class GdsApi::PublishingApiV2 < GdsApi::Base
+  # Put a content item
+  #
+  # This creates a new draft item, which will be sent to the draft content store.
+  #
+  # @param content_id [UUID]
+  # @param payload [Hash] A valid content item
+  #
+  # @see https://github.com/alphagov/publishing-api/blob/master/doc/publishing-api-syntactic-usage.md#put-v2contentcontent_id
```

Comment:
This is good to link to. There's also https://github.com/alphagov/publishing-api/blob/master/doc/publishing-application-examples.md
and https://github.com/alphagov/publishing-api/blob/master/doc/object-model-explanation.md which I've found useful. Do you think it makes sense to link these anywhere here?